### PR TITLE
Javascript functionality

### PR DESCRIPTION
**Pull request by LL835**
Hey team, here's what I've managed to do so far:

- Added to Georgia's code to add and remove tasks from the DOM / tasks array.
- Added checkbox and remove buttons.
- Adjusted the media queries.

Lmk how it looks. Change and remove whatever you want too.

### CODE REVIEW

File: scripts/script.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+let listArray = [];
+
+const form = document.querySelector(".form");
+const input = form.querySelector(".form-input");
+const list = document.querySelector(".list");
+
+form.addEventListener("submit", (x) => {
```
Comment (georgiaewhitney) [inline scripts/script.js:7]: Karol left us a helpful note to change these to e/event - which have done throughout - thanks! I'll change this one now.

**Review by georgiaewhitney — APPROVED** [review]
Hey @LL835 / Derek - this is looking great! Perfect changes, and you've tidied up the script, hitting Karol's code review the other day too. There's one x remaining, I'll remove this.

File: scripts/script.js (hunk 1 above)
Comment (LL835) [reply scripts/script.js:7]: Thanks, Georgia!

Comment (DominicSimpson) [reply scripts/script.js:7]: Thank you, yes it should usually be either e or event.

**Comment by DominicSimpson** [comment]
Thank you so much for this - it's fantastic. I'll add a footer tomorrow. I really appreciate the work that you've put into this.

**Pull request merged**